fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -796,8 +852,10 @@ export class PseudoFWC {
         this.engine2State.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_STATE:2', 'Enum'));
         this.N1Eng1.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N1:1', 'number'));
         this.N1Eng2.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N1:2', 'number'));
-        this.N1IdleEng1.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_IDLE_N1:1', 'number'));
-        this.N1IdleEng2.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_IDLE_N1:2', 'number'));
+        this.N2Eng1.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N2:1', 'number'));
+        this.N2Eng2.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N2:2', 'number'));
+        this.N1IdleEng.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_IDLE_N1', 'number'));
+        const oneEngineAboveMinPower = this.engine1AboveIdle.get() || this.engine2AboveIdle.get();
```

Comment:
After havingg seen this on a stream just now: I think this is missing a comparison, like >= 20 (you gotta figure out a good N1 threshold somewhere).

Reply:
Yeah, I noticed that. The current comparison is using a threshold of 0.5 above the idle N1… but the engines actually seem to sit up to 1 above idle N1 in descent.

Reply:
Really the ECU should calculate this anyway.